go/cmd/vtorc/main.go

```diff
@@ -151,9 +152,9 @@ Please update your scripts before the next version, when this will begin to brea
 		log.Warningf(warning, args, os.Args)
 	}
 
-	_flag.Parse(fs)
+	servenv.ParseFlags("vtorc")
 	// N.B. Also duplicated from go/internal/flag, for the same reason as above.
-	if help || fs.Arg(0) == "help" {
+	if help {
 		pflag.Usage()
 		os.Exit(0)
 	}
```

Comment:
I checked removing this shim code. Then `--help` and `-h` work, but `-help` does not. 
In `transformArgsForPflag`, we don't change flags that haven't been registered yet. Like 
```go
if fs.Lookup(name) != nil {
				// Case 1: We have a long opt with this name, so we need to
				// prepend an additional hyphen.
				result = append(result, "-"+arg)
			} else {
				// Case 2: No transformation needed.
				result = append(result, arg)
			}
```
What this means is that we don't change `-help` to `--help` and then pflag complains with unknown shorthand `-e`.
So we need to define the `help` flag before we transform, but if we define the flag ourselves beforehand, then the defer also doesn't work, so we need to catch this too here.

Does it make sense?

Reply:
i like your alternate (explicitly checking for the literal `"help"`); since we're not doing this generally but for `vtorc` specifically that seems totally fine to me

Reply:
Changed it 👍. Thank-you for the help with the flag changes! ❤️